airbyte-integrations/connectors/source-netsuite/source_netsuite/source.py

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from typing import Any, List, Mapping, Tuple
+
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from netsuitesdk import NetSuiteConnection
+
+from source_netsuite.streams import (
+    Accounts,
+    Classifications,
+    Currencies,
+    Customers,
+    Departments,
+    Employees,
+    ExpenseCategories,
+    ExpenseReports,
+    Files,
+    Folders,
+    JournalEntries,
+    Locations,
+    Projects,
+    Subsidiaries,
+    TaxGroups,
+    TaxItems,
+    Terms,
+    VendorBills,
+    VendorPayments,
+    Vendors,
+)
+
+
+class SourceNetsuiteSoap(AbstractSource):
+    @staticmethod
+    def get_netsuite_connector(config: Mapping[str, Any]) -> NetSuiteConnection:
```

Comment:
Let's rename this to `get_authenticator`

Reply:
Generally speaking, this method is not needed here, since we are using external dependency that could be called directly using just mapping, like:
```python
config_map = {
    "account": config["realm"],
    "consumer_key": config["consumer_key"],
    "consumer_secret": config["consumer_secret"],
    "token_key": config["token_id"],
    "token_secret": config["token_secret"],
}
```

and call the `NetSuiteConnection` dicrectly, using:

```python
auth = NetSuiteConnection(**config_map)
```

Reply:
Generally speaking, this method is not needed here, since we are using external dependency that could be called directly using just mapping, like:
```python
config_map = {
    "account": config["realm"],
    "consumer_key": config["consumer_key"],
    "consumer_secret": config["consumer_secret"],
    "token_key": config["token_id"],
    "token_secret": config["token_secret"],
}
```
and call the `NetSuiteConnection` dicrectly, using:
```python
auth = NetSuiteConnection(**config_map)
```